x-pack/plugins/security/server/routes/views/capture_url.ts

```diff
@@ -16,13 +16,7 @@ export function defineCaptureURLRoutes({ httpResources }: RouteDefinitionParams)
   httpResources.register(
     {
       path: '/internal/security/capture-url',
-      validate: {
-        query: schema.object({
-          providerType: schema.string({ minLength: 1 }),
-          providerName: schema.string({ minLength: 1 }),
-          next: schema.maybe(schema.string()),
-        }),
-      },
+      validate: { query: schema.object({ next: schema.maybe(schema.string()) }) },
```

Comment:
question: is there any chance this change could break us? I don't have a scenario in mind (other than stale client code), but I get nervous when we touch the auth flow. Would it make sense to add `{ unknowns: 'ignore' }`, or am I just being paranoid?

Reply:
> but I get nervous when we touch the auth flow. 

<p align="center">
  <img src="https://user-images.githubusercontent.com/1713708/114686050-b5ccc480-9d12-11eb-9ba3-ce113f45cdf6.png">
</p>

But all jokes aside, `{ unknowns: 'ignore' }` wouldn't hurt, will add it, thanks!